src/common/RAJAPerfSuite.hpp

```diff
@@ -16,16 +16,16 @@
 #include "RAJA/config.hpp"
 #include "rajaperf_config.hpp"
 
-#include <string>
-#include <ostream>
+#if defined(RUN_KOKKOS)
+#include "Kokkos_Core.hpp"
+#endif // RUN_KOKKOS
```

Comment:
This is a high level header and based on the changes I'm skeptical that it needs to include kokkos. Was kokkos included here to avoid including it elsewhere?